### DISCUSSION TITLE
Is It Down?

### DISCUSSION THREAD

**Discussion by khushalmajoka** [Algolisted-Org/AlgoListed#270] [General]
I don't know what's the issue, but is Algolisted.com is down?

**Comment by NayakPenguin** [comment]
Thanks for bringing it to hour notice, we are looking into it.

**Comment by NayakPenguin** [comment]
@khushalmajoka google firebase is down, unfortunately we can't do anything about it :)

**Comment by khushalmajoka** [comment]
Ohh it was the issue then, i though I have some network issue, so connected to the VPN and it did work, so I thought I unnecessarily raised the issue 😂😂, It was a real issue then.
